build.gradle.kts

```diff
@@ -486,6 +486,8 @@ allprojects {
             (options as StandardJavadocDocletOptions).apply {
                 // Please refrain from using non-ASCII chars below since the options are passed as
                 // javadoc.options file which is parsed with "default encoding"
+                // locale should be placed at the head of any options: https://docs.gradle.org/current/javadoc/org/gradle/external/javadoc/CoreJavadocOptions.html#getLocale
+                locale = "en_US"
```

Comment:
Nice finding
Am I correct that same thing should be done for Avatica?

Reply:
Yes, I do think so. I'll do this for avatica once this is merged.

Reply:
The PR against avatica is now available for review: https://github.com/apache/calcite-avatica/pull/235